### PR TITLE
Add koala backend link

### PR DESCRIPTION
**Pull request by 3eak**
This PR aims to add a link to the backend bot, so that we can call commands from the backend in the frontend cog pages.

### CODE REVIEW

File: requirements.txt [hunk 1]
```diff
@@ -3,3 +3,33 @@ Python-dotenv~=0.15.0
...
 whitenoise==5.2.0
+discord.py
+aiohttp==3.6.2
+async-timeout==3.0.1
+atomicwrites==1.4.0
+attrs==20.1.0
+certifi==2020.6.20
+chardet==3.0.4
+colorama==0.4.3
+dpytest==0.0.20
```
Comment (JayDwee) [inline requirements.txt:14]: unnecessary

File: frontend/views.py [hunk 2]
```diff
@@ -1,6 +1,28 @@
...
 
+import KoalaBot
+from KoalaBot import database_manager
+from cogs import BaseCog
+from cogs import TextFilter
+from cogs.TextFilter import TextFilterDBManager
+
+# exampleBackend()
+# This is only an example to document how you might, we can delete this once we have some cog stuff in
+def exampleBackend():
```
Comment (JayDwee) [inline frontend/views.py:12]: I assume you've tested this? in which case it looks like it'll work great!

File: .gitignore [hunk 3]
```diff
@@ -128,3 +128,5 @@ dmypy.json
 
 # Pyre type checker
 .pyre/
+
+Koala.db
```
Comment (JayDwee) [inline .gitignore:132]: more database names are used
```suggestion
*.db
```

**Review by JayDwee — CHANGES_REQUESTED** [review]
.gitignore is the main thing to change. The rest look great

File: frontend/views.py (hunk 2 above)
Comment (3eak) [reply frontend/views.py:12]: yep - it's just a very small test case to show that we can add and read something from a database. 

i'm gonna add the imports in there too actually to make it clear which things are needed.

**Review by JayDwee — APPROVED** [review]

**Pull request merged**